setup.py

```diff
@@ -10,5 +10,6 @@
       include_package_data=True,
       description=('Parameter Manager for LCLS Device Configurations'),
       entry_points={'console_scripts': ['pmgr = pmgr.pmgr:main',
+	                                'fixup = pmgr.migrate.fixup:main',
```

Comment:
Do you intend for the `fixup` script to be bound when people activate the `pcds-conda`?

Reply:
Not really.  The fixup script is to be run once, when moving from the Old Regime to the Brave New World.  It was just convenient to make it an entry point here.  Once we are ready to go, I can run the script, remove this, then check everything in and do a release.

Reply:
Removing prior to releasing seems reasonable.

At the very least prefixing by `pmgr-` to make it `pmgr-fixup` instead of the overly general `fixup` would be advisable.

Reply:
I've added pmgr- as a prefix.